include/bluetooth/services/ots.h

```diff
@@ -548,6 +548,31 @@ struct bt_ots_cb {
 	uint32_t (*obj_read)(struct bt_ots *ots, struct bt_conn *conn,
 			     uint64_t id, uint8_t **data, uint32_t len,
 			     uint32_t offset);
+
+	/** @brief Object write callback
```

Comment:
It is unclear from this function if the application has to verify the offset and length attributes, (and what the application should verify if any) from this function.
By looking at the code it seems that the stack verifies length and offset.

I think it would make sense to document this.

Reply:
I have updated the function documentation to reflect the fact that the OTS module validates that the offset and length are within the allocated object size.